packages/pharos/src/components/dropdown-menu/pharos-dropdown-menu-item.test.ts

```diff
@@ -110,4 +110,17 @@ describe('pharos-dropdown-menu-item', () => {
     component.dispatchEvent(event);
     expect(clickSpy.callCount).to.equal(1);
   });
+
+  it('do not trigger click event when disabled and @click handler present', async () => {
```

Comment:
```suggestion
  it('does not trigger a click event when disabled with click handler present', async () => {
```

Reply:
@daneah  Thank you for the clarification. Just added check for `stopPropagation` as well.